docs/editor_integration.md

```diff
@@ -233,6 +233,32 @@ If you later want to update _Black_, you should do it like this:
 $ pip install -U black --no-binary regex,typed-ast
 ```
 
+## Gedit
+gedit is the default text editor of the GNOME, Unix like Operating Systems.
+Open gedit as
+```console
+$ gedit <file_name>
+```
+1. `Go to edit > preferences > plugins`
+2.  Search for `external tools` and activate it.
+3.  In `Tools menu -> Manage external tools`
+4.  Add a new tool using `+` button.
+5.  Copy the below content to the code window.
+```console
+#!/bin/bash
+Name = $GEDIT_CURRENT_DOCUMENT_NAME
+black $Name
+```
+- Set a keyboard shortcut if you like, Ex. `ctrl-B`
+- Save: `Nothing`
```

Comment:
I really hope that this doesn't turn into a source of "I JUST LOST A BUNCH OF MY WORK AND I BLAME YOU" type of bug reports, but I don't use gedit so I have no idea.